imgp

```diff
@@ -449,6 +451,8 @@ def parse_args(args=None, namespace=None):
            help='adapt to resolution by orientation [default: off]')
     addarg('-c', '--convert', action='store_true',
            help='convert PNG to JPG format [default: off]')
+    addarg('-nn', '--nearest-neighbour', action='store_true',
+           help='If the image is a png, use nearest neighbour interpolation [default: off]')
```

Comment:
Please change help to:

`use nearest neighbour interpolation for PNG images [default: off]`

and place it between options `-n` and `-p` (I tend to maintain the lower portion alphabetically)

Reply:
Please remove '--nearest-neighbour'. No one is going to type that ever. Help says all.